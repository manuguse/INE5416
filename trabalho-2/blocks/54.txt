1 2 7 7 12 13 18 21 21 21
1 2 7 8 13 13 18 21 21 21
2 2 8 8 13 13 19 19 19 19
2 2 8 8 13 13 16 16 23 23
3 3 9 9 9 16 16 16 23 25
4 3 3 3 9 9 20 17 23 25
4 4 4 3 11 9 9 17 23 23
4 6 6 11 11 17 17 17 24 24
4 4 10 10 14 14 15 22 22 22
5 5 10 10 15 15 15 15 22 22