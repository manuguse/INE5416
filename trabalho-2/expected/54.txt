2 6 3 2 1 7 2 5 6 3
1 5 1 4 5 6 1 4 1 2
2 4 5 3 4 2 3 2 4 1
1 3 2 1 3 1 5 4 6 5
6 5 1 6 4 2 3 1 3 2
5 3 2 4 3 7 1 5 2 1
3 7 4 1 2 5 2 4 1 4
2 1 2 3 1 2 1 3 2 1
1 6 3 4 2 1 3 1 5 3
2 1 2 1 5 4 1 2 4 2